Xamarin.Forms.Platform.GTK/Renderers/BoxViewRenderer.cs

```diff
@@ -1,85 +1,97 @@
 ﻿using System.ComponentModel;
+using Xamarin.Forms.Core;
```

Comment:
need to drop this. this namespace doesn't exist

Reply:
Also about this renderer, I see a white background , can you confirm that if we set the color of background to Transparent it does not show the white background.

<img width="427" alt="screen shot 2018-03-22 at 15 03 47" src="https://user-images.githubusercontent.com/1235097/37778625-514fb6ae-2de2-11e8-834d-683f4d12fb45.png">